src/components/DeleteMediaModal/DeleteMediaModal.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import { useDisclosure } from "@chakra-ui/react"
+import { Button } from "@opengovsg/design-system-react"
+import type { Meta, StoryFn } from "@storybook/react"
+
+import { getMediaLabels } from "utils/media"
+
+import { useSuccessToast } from "utils"
+
+import { DeleteMediaModal } from "./DeleteMediaModal"
+
+const deleteMediaModalMeta = {
+  title: "Components/Delete Media Modal",
+  component: DeleteMediaModal,
+} as Meta<typeof DeleteMediaModal>
+
+const deleteMediaModalTemplate: StoryFn<typeof DeleteMediaModal> = ({
+  selectedMedia,
+}) => {
+  const { isOpen, onOpen, onClose } = useDisclosure({ defaultIsOpen: true })
+  const successToast = useSuccessToast()
+  const onProceed = () => {
+    successToast({
+      id: "storybook-delete-media-success",
+      description: "STORYBOOK: Media has been successfully deleted",
+    })
+    onClose()
+  }
+
+  return (
+    <>
+      <Button onClick={onOpen}>Open delete media modal</Button>
+      <DeleteMediaModal
+        selectedMedia={selectedMedia}
+        mediaLabels={getMediaLabels("images")}
+        isWriteDisabled={false}
+        isOpen={isOpen}
+        isLoading={false}
+        onClose={onClose}
+        onProceed={onProceed}
+      />
+    </>
+  )
+}
+
+export const Default = deleteMediaModalTemplate.bind({})
+Default.args = {
+  selectedMedia: [
+    { filePath: "/images/hero-banner.png", size: 1234, sha: "sha1234" },
+  ],
+}
+
+export const MultipleMedia = deleteMediaModalTemplate.bind({})
+MultipleMedia.args = {
+  selectedMedia: [
+    { filePath: "/images/hero-banner.png", size: 1234, sha: "sha1234" },
```

Comment:
lets add units please + round to nearest number 2sf?

Reply:
Hmm it's supposed to be random data, not supposed to mean anything.